src/libcore/cmp.rs

```diff
@@ -95,6 +96,76 @@ use self::Ordering::*;
 /// assert!(b1 != b3);
 /// ```
 ///
+/// ## How can I implement it to compare with another type?
+///
+/// The type you can compare with is controlled by `PartialEq`'s type parameter.
+/// For example, let's tweak our previous code a bit:
+///
+/// ```
+/// enum BookFormat { Paperback, Hardback, Ebook }
```

Comment:
is this how `rustfmt` formats this code?

Reply:
I took the code above as is so I don't know?